resources/assets/js/components/privacy-info/PrivacyInfo.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <privacy-info-logic>
+        <div class="block" v-if="toShow" slot-scope="{ toShow, accept }">
+            <div class="message">
+                Dostosowaliśmy naszą Politykę Prywatności do nowych wymogów rozporządzenia Unii Europejskiej o ochronie
```

Comment:
Please use translations for all the text in this component and provide an English translation